tests/modules/images/webp-uploads/helper-tests.php

```diff
@@ -367,7 +367,7 @@ public function it_should_return_default_transforms_when_filter_returns_non_arra
 		add_filter(
 			'webp_uploads_upload_image_mime_transforms',
 			function () {
-				return;
+				return null;
 			}
 		);
```

Comment:
Should we use the `__return_null` function here instead of adding this function?
```php
add_filter( 'webp_uploads_upload_image_mime_transforms', '__return_null' );
```

Reply:
Done in 93ec3dce1038874f539ff6e0f15c636304ebb3d2